src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/train/variable_speed_compressor_train_common_shaft_multiple_streams_and_pressures.py

```diff
@@ -769,15 +769,18 @@ def evaluate_rate_ps_pint_pd(
                 train_results.append(compressor_train_result)
 
         power_mw = np.array([time_step.power_megawatt for time_step in train_results])
+        power_mw_adjusted = np.where(
+            power_mw > 0, power_mw + self.data_transfer_object.energy_usage_adjustment_constant, power_mw
```

Comment:
```suggestion
            power_mw > 0, power_mw + self.energy_usage_adjustment_constant, power_mw
```
In my opinion I am not fan of the DTO layer not being contained in the initialisation of objects.

Suggest creating 'self.data_transfer_object.energy_usage_adjustment_constant' in the init method?

Reply:
I agree, but this has been violated all over the place already ..